CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # Changelog
+## [v1.14.2](https://github.com/contentstack/contentstack-management-javascript/tree/v1.14.2) (2024-11-23)
```

Comment:
@sunil-lakshman Please add the Changelog entries for all the versions between 1.13.1 and 1.14.2

Reply:
Added @nadeem-cs